demos/asp-net-core/ITOps.ViewModelComposition/IViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition
+{
+    public interface IViewModel
```

Comment:
It has no real usage, it use Udi that wanted it to mark that a dynamic ViewModel is a `IViewModel`, the idea was that appenders could have had their own interface inheriting from `IViewModel` to express the intent of using a concrete type instead of `dynamic`. I'm 👍 to drop it since we're not using it at all